main.go

```diff
@@ -102,308 +124,285 @@ func exportEnvironmentWithEnvman(keyStr, valueStr string) error {
 	return cmd.Run()
 }
 
-// -----------------------
-// --- Main
-// -----------------------
-func main() {
-	//
-	// Validate options
-	configs := createConfigsModelFromEnvs()
-
-	fmt.Println()
-	configs.print()
+func run(c *command.Model) error {
```

Comment:
move the git related commands to a separate file for cleaner code (`SOURCE_DIR/git.go`)